### DISCUSSION TITLE
Problem with `'vite-plugin-ssr' module or its type declaration not found.ts(2307)` error after update

### DISCUSSION THREAD

**Discussion by moonhee0507** [vikejs/vike#785] [Discussion] [answered]
I have Updated multiple packages including vite-plugin-ssr with `npm-check-updates`.

But after update
In the types.ts file, I am getting the `'vite-plugin-ssr' module or its type declaration not found.ts(2307)` error.

It seems that there was a file related to type in the original package, but is it true that it is gone?
What did I do wrong? I just wanted to write `routeParams` after updating..!

[Here is my Repo](https://github.com/moonhee0507/green-maps/blob/main/renderer/types.ts)

<img width="407" alt="image" src="https://user-images.githubusercontent.com/102460056/231873938-c6f5c60c-64bd-491d-b4b4-5d4eb586a68a.png">

<img width="104" alt="image" src="https://user-images.githubusercontent.com/102460056/231874013-442d4c36-370c-40e4-8a15-e3bdaec8e190.png">

**Comment by brillout** [comment]
Import them from `vite-plugin-ssr/types`.

(I'll have a look at why the deprecation warning doesn't seem to have worked in your reproduction.)

**Reply by brillout** [reply]
> (I'll have a look at why the deprecation warning doesn't seem to have worked in your reproduction.)

Fix released in `0.4.113`.

**Reply by brillout** [reply, marked as answer]
> Fix released in `0.4.113`.

You should now get a warning instead of an error.

**Reply by moonhee0507** [reply]
> > Fix released in `0.4.113`.
> 
> You should now get a warning instead of an error.

The errors about types no longer show. Thank you for your quick response!

**Reply by brillout** [reply]
Thanks for the bug report!

Btw. in case you think you can convince your employer: https://github.com/sponsors/brillout. Sponsors are starting to make a big difference.
